lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -59,7 +59,7 @@ module.exports = [
         extendedInfo: {
           value: {
             results: {
-              length: 3
+              length: 4
```

Comment:
we actually had 4 failures before, just 2 of them were de-duped with the same URL, now that I added different delays it should be 4.